Mopups/Mopups.Maui/Platforms/Android/Impl/AndroidMopups.cs

```diff
@@ -70,7 +89,44 @@ public Task RemoveAsync(PopupPage page)
         return Task.CompletedTask;
     }
 
-    Task<bool> PostAsync(Android.Views.View nativeView)
+    static void HandleAccessibility(ImportantForAccessibility importantForAccessibility)
+    {
+        Page? mainPage = Application.Current?.MainPage;
+
+        if (mainPage is null)
+        {
+            return;
```

Comment:
How often is it that the MainPage is null? Do you think this should be a silent return, or an exception thrown?

Reply:
None that I have come across. 

But checking for null at that point means the nav count variables don't become nullable `(int?)` meaning we don't have to cast them to an int when getting the page from the stack

Reply:
Ah excellent point 👍🏻 just am wary of early returns. All good